pkg/ansible/paramconv/paramconv.go

```diff
@@ -70,6 +70,52 @@ func ToCamel(s string) string {
 	return ret
 }
 
+// preprocessWordMapping() will check if value contains special words mapped or its plural in
+// wordMapping, then processes it such that ToSnake() can convert it to snake case.
+// If value contains special word, the character "_" is appended as a prefix and postfix
+// to the special word found. For example, if the input string is "egressIP",
+// which contains is a special word "IP", the function will return "egress_IP".
+// If the last character of the special word is an "s" (i.e plural of the word
+// found in wordMapping), it is considered a part of that word and will be capitalized.
+func preprocessWordMapping(value string) string {
```

Comment:
Can a value contain multiple special words?

Reply:
No it does not address multiple special words. Would you want me to add that feature?

Reply:
@fabianvf can an ansible variable name contain multiple special words?

Reply:
Yeah, can be any arbitrary string